apps/syntrest/README.md

```diff
@@ -110,22 +110,22 @@ De filene som må endres når man legger til synt-pakkene er:
 Oppdater `kube-config-path` i `application-dev.properties` til din lokale kubeconfigs path.
 Kjør så ApplicationStarter med følgende argumenter:
 
+### I utviklerimage
 ```
  -Djavax.net.ssl.trustStore=[path til lokal truststore]
  -Djavax.net.ssl.trustStorePassword=[passord til lokal truststore]
  -Dspring.profiles.active=dev
  -DGITHUB_USERNAME=[github brukernavn, kan finnes i syntrest secrets i vault]
  -DGITHUB_PASSWORD=[github passord, kan finnes i syntrest secrets i vault]
+ -Dproxy-url=webproxy-utvikler.nav.no
+ -Dproxy-port=8088
 ```
-Utenfor utviklerimage er det unødvendig med `trustStore`.
-Hvis ApplicationStarter ikke vil kjøre sjekk at du fremdeles har gyldig Kubectl token (`kubectl get pods` kommando fungerer lokalt.)
-
-### Image tag - Deploye synt-applikasjoner lokalt
+#### Image tag - Deploye synt-applikasjoner lokalt
 Når syntrest kjøres lokalt så klarer den ikke alltid å hente korrekt/siste image tag fra github for synt-applikasjon som skal deployes. 
 En manuell fiks for dette er å gå til [packages](https://github.com/orgs/navikt/packages?repo_name=testnorge-syntetiseringspakker) og hente siste versjon for ønsket 
 applikasjon og bytte ut `latest` med denne versjonen i image-delen i relevant nais.yaml (resources/nais/{app-name}.yaml). Deretter må følgende kode kommenteres ut i 
 `KubernetesController` `prepareYaml`-metode:
-```
+```java
```

Comment:
Skal det stå java her?

Reply:
Er for context highlighting på github ;)

Reply:
Ok, jeg fjernet den nå når jeg la til ting i readme, så det er bare å sette den tilbake.

Reply:
Fikset det selv nå ;)